docs/guides/get-started-with-primitives.mdx

```diff
@@ -73,7 +73,7 @@ isa_observable = observable.apply_layout(isa_circuit.layout)
 
 ### 3. Initialize Qiskit Runtime Estimator
 
-When you initialize the Estimator, pass in the system or simulator you previously selected as the target.
+When you initialize the `Estimator`, use the `mode` parameter to specify the mode you want it to run in.  Possible values are `batch`, `session`, or `backend` for job execution mode. For more information, see [Introduction to Qiskit Runtime execution modes.](execution-modes)
```

Comment:
I'd like to point out `batch` etc are name of the objects, not literally "batch". 

```suggestion
When you initialize the `Estimator`, use the `mode` parameter to specify the mode you want it to run in.  Possible values are `batch`, `session`, or `backend` objects for batch, session, and job execution mode, respectively.. For more information, see [Introduction to Qiskit Runtime execution modes.](execution-modes)
```